PENDING_PENDING_RELEASE_NOTES.md

```diff
@@ -2,6 +2,9 @@
 ## Version 16.2.1
```

Comment:
If the PENDING_RELEASE_NOTES version is at 16.2.0, and this PR now has a new Feature, then the new version should be 16.3.0, not 16.2.1, which would indicate it was only a bugfix